config/logstash.yml

```diff
@@ -98,13 +98,12 @@
 # ------------ Debugging Settings --------------
 #
 # Options for log.level:
-#   * warn => warn (default)
-#   * quiet => error
-#   * verbose => info
-#   * debug => debug
+#   * warn (default)
+#   * error
```

Comment:
Nice :)
In every conf/code listing all log levels, it would be nice to add `fatal` and `trace` levels, as you implemented these levels in `logger.rb`